src/main/java/bio/terra/app/model/GoogleRegion.java

```diff
@@ -6,51 +6,61 @@
  * Valid regions in Google.
  */
 public enum GoogleRegion {
-        ASIA_EAST1("asia-east1"),
         ASIA_EAST2("asia-east2"),
+        ASIA_EAST1("asia-east1", GoogleRegion.ASIA_EAST2, null),
```

Comment:
Note that this kind of cross reference between enum instances has some limitations. Since you can't ensure that one enum instance has been initialized before another, it's not safe to refer to the contents 
(members) of an enum instance inside the instance constructor. Referring to the instance itself should be ok though.